charts/prometheus-statsd-exporter/values.yaml

```diff
@@ -63,18 +63,49 @@ readinessProbe:
     path: /health
     port: http
 
-serviceMonitor:
```

Comment:
Please keep the current defaults. Otherwise it breaks for everyone updating from earlier versions.

Please also keep Servicemonitor and PodMonitor config separated.

Reply:
I am a bit confused between your first comment and this one.
so I update the PR and hope I got your point correctly.

Otherwise, could you please provide more context of your expectation on supporting Google Managed Prometheus config for this helm chart.